toolkits/jira/arcade_jira/tools/users.py

```diff
@@ -56,9 +66,15 @@ async def list_users(
 async def get_user_by_id(
     context: ToolContext,
     user_id: Annotated[str, "The the user's ID."],
+    atlassian_cloud_id: Annotated[
```

Comment:
don't forget to provide this to the client call on line 82

Reply:
It's provided on line 77. It's not necessary to provide to the `get` method. The URL is built using the internal `cloud_id` property.